jack-and-the-beans/buddies-ios
# Fix issues with activity table

This PR:

- Fixes #113
- Fixes #87
- Fixes #56
- Fixes #97
- Fixes #111
- Fixes #117
- Fixes #118
- [EDIT: not #119, but it rarely happens because other bugs are fixed]
- Adds "pull to refresh" to the activity table
- Shows activity indicator while results are loading

### To Test:
- Run the app and examine all of the activity tables
- Make sure that updates and deletions to activities properly refresh on the tableview in realtime.
- Make sure that there is no inconsistency with user pictures - the ones on the table should be the same as in view Activity.
- Make sure that the user experience isn't jarring - we don't want to have a lot of flashes and stuff.
